domains/chrisnguyen.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "chrisnguyen.is-a.dev : This is my personal website",
+  "owner": {
+    "username": "chrisnguyen",
```

```suggestion
    "username": "chris-thegamerchanger",
```
This must be your GitHub username.